packages/template-retail-react-app/app/components/search/index.jsx

```diff
@@ -159,6 +198,24 @@ const Search = (props) => {
         if (searchText.length < 1) {
             return
         }
+
+        if (askAgentOnSearchEnabled && window.embeddedservice_bootstrap) {
+            // Add a 500ms delay before sending the message to ensure the experience isn't jarring to the user
+            setTimeout(() => {
+                window.embeddedservice_bootstrap.utilAPI
+                    .sendTextMessage(searchText)
+                    .catch((err) => {
```

Comment:
Let's print out the error for easier debugging.

```suggestion
                    .catch((err) => {
                        console.err(err)
```